dist.java/java/404.html

```diff
@@ -38,10 +38,10 @@ <h4>Connected to <span ng-bind="containerName"></span></h4>
       <div ng-include src="viewPartial"></div>
     </div>
 
-    <script src="libs-301d12fa8a.js"></script>
+    <script src="libs-3239db0851.js"></script>
```

Comment:
Looks like you'll pulling in an old JVM console. You'll want to run

```
$ hack/clean-deps.sh && hack/install-deps.sh && grunt build
```

Please squash commits when finished :)

Reply:
Thanks! That looks like it fixed it up